core/src/main/java/com/nikodoko/javaimports/cli/UsageException.java

```diff
@@ -21,6 +22,8 @@ public class UsageException extends Exception {
     "    Print the version.",
     "  --help, -help, -h",
     "    Print this usage statement.",
+    "  --assume-filename, -assume-filename",
```

Comment:
Could you put the option above `--fix-only`? I'm trying to have options in alphabetical order, with help as the last one.